README.md

```diff
@@ -80,7 +80,110 @@ export TORCH_SENDNN_LOG=CRITICAL
 export DT_DEEPRT_VERBOSE=-1
 ```
 
-## Example runs
+## Example of using Foundation Model Stack (FMS) on AIU hardware
+
+Let's look at a simple example of running [FMS](https://github.com/foundation-model-stack/foundation-model-stack) on AIU. While you can use a variety of models, in this example, we will use the Llama-194m decoder model.
+
+The first step is to make sure that the required libraries are available. For our example, we will use the following libraries.
```

Comment:
say the required libraries are torch, hf transformers, FMS and AFTU and link them

Reply:
Sure!